packages/fluentui/react-northstar/src/components/Popup/PopupContent.tsx

```diff
@@ -59,82 +75,123 @@ export interface PopupContentProps extends UIComponentProps, ChildrenComponentPr
   autoFocus?: boolean | AutoFocusZoneProps;
 }
 
-class PopupContent extends UIComponent<WithAsProp<PopupContentProps>> {
-  static create: ShorthandFactory<PopupContentProps>;
-
-  static displayName = 'PopupContent';
-  static className = 'ui-popup__content';
-  static slotClassNames: PopupContentSlotClassNames;
-
-  static propTypes = {
-    ...commonPropTypes.createCommon(),
-    placement: PropTypes.string,
-    pointing: PropTypes.bool,
-    onMouseEnter: PropTypes.func,
-    onMouseLeave: PropTypes.func,
-    pointerRef: customPropTypes.ref,
-    trapFocus: PropTypes.oneOfType([PropTypes.bool, PropTypes.object]),
-    autoFocus: PropTypes.oneOfType([PropTypes.bool, PropTypes.object]),
-  };
+export type PopupContentStylesProps = Pick<PopupContentProps, 'placement' | 'pointing'>;
+
+const PopupContent: React.FC<WithAsProp<PopupContentProps>> &
+  FluentComponentStaticProps<PopupContentProps> & { slotClassNames: PopupContentSlotClassNames } = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext);
+  const { setStart, setEnd } = useTelemetry(PopupContent.displayName, context.telemetry);
+  setStart();
 
-  handleMouseEnter = e => {
-    _.invoke(this.props, 'onMouseEnter', e, this.props);
+  const {
+    accessibility,
+    autoFocus,
+    children,
+    className,
+    content,
+    design,
+    pointing,
+    pointerRef,
+    styles,
+    trapFocus,
+    variables,
+  } = props;
+
+  const getA11yProps = useAccessibility(accessibility, {
+    debugName: PopupContent.displayName,
+    rtl: context.rtl,
+  });
+  const { classes } = useStyles<PopupContentStylesProps>(PopupContent.displayName, {
+    className: PopupContent.className,
+    mapPropsToStyles: () => ({
+      pointing,
```

Comment:
```suggestion
      pointing,
      placement,
```

Shouldn't we pass placement as well, since it's used in packages/fluentui/react-northstar/src/themes/teams/components/Popup/popupContentStyles.ts ?

Reply:
Good catch, thank you